src/styles/global.css

```diff
@@ -48,6 +48,8 @@
       rgba(75, 231, 156, 0.1) 44.21%,
       rgba(231, 202, 200, 0.1) 82.88%
     );
+
+    --gradient-step-1: rgba(127, 127, 213, 0.20);
```

Comment:
why is it call `step-1`? looking into the code, this looks like a new card or banner gradient, I'd align its name with the other similar gradients, `--banner-secondary-gradient` or `--card-secondary-gradient`.

Also, I'd encourage to reuse some of the already existing gradients cc @nloureiro @konopkja